openlibrary/templates/search/sort_options.html

```diff
@@ -0,0 +1,28 @@
+$def with (exclude, default)
```

Comment:
You have to actually _use_ `exclude` somewhere; it's not there just to look pretty :P You'll likely want to in the for loop do an `if exclude and sort_option['sort'] in exclude: continue`.